mysite/code/dataobjects/Addon.php

```diff
@@ -25,6 +25,8 @@ class Addon extends DataObject
         // Module rating information
         'Rating'            => 'Int',
         'RatingDetails'     => 'Text',
+        // Supported by SS
```

Comment:
Can you clarify this comment to say something like "Commercially supported by SilverStripe Ltd" ?

Reply:
👍